From 4ce7f1c9eb490208e36b1910845dd1f577f639a5 Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Sun, 26 Aug 2012 18:09:38 -0500
Subject: [PATCH 048/602] ENGR00215487: Default BOARD_WLAN_VENDOR=ATHEROS for
 iMX53 SMD

Default BOARD_WLAN_VENDOR to ATEHROS for iMX53 SMD.

Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 imx53_smd/BoardConfig.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx53_smd/BoardConfig.mk b/imx53_smd/BoardConfig.mk
index 652e608..d0ba255 100755
--- a/imx53_smd/BoardConfig.mk
+++ b/imx53_smd/BoardConfig.mk
@@ -10,6 +10,7 @@ BOARD_SOC_CLASS := IMX5X
 BOARD_SOC_TYPE := IMX53
 
 # Wifi
+BOARD_WLAN_VENDOR := ATHEROS
 BOARD_WLAN_DEVICE := ar6003
 BOARD_HAS_ATH_WLAN := true
 BOARD_WLAN_ATHEROS_SDK := system/wlan/atheros/compat-wireless
-- 
1.8.0

